src/contactsui.cs

```diff
@@ -74,6 +77,7 @@ interface CNContactPickerDelegate
 		void DidClose (CNContactPicker picker);
 	}
 #else
+	[NoMac]
```

Comment:
Any reason you didn't merge the `CNContactPickerDelegate` definitions like you did for `CNContactViewController`?

Reply:
This was just found later and I started running into more time pressure to get this done.